packages/block-library/src/breadcrumbs/index.php

```diff
@@ -136,7 +136,7 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 		}
 
 		// Add post title: linked when viewing a paginated page, plain text otherwise.
-		$is_paged = (int) get_query_var( 'page' ) > 1;
+		$is_paged = (int) get_query_var( 'page' ) > 1 || (int) get_query_var( 'cpage' ) > 1;
```

Comment:
Given that we are updating the is_paged flag should we also update the function block_core_breadcrumbs_is_paged.